carma-messenger-config/chevrolet_tahoe_2018/docker-compose-background.yml

```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   carma-messenger-ui:
-    image: ${DOCKER_ORG}/carma-messenger-ui:${DOCKER_TAG}
+    image: usdotfhwastolcandidate/carma-messenger-ui:prowler
```

Comment:
Instead of hardcoding this, we can just set the DOCKER_ORG and DOCKER_TAG variables to the required values

Reply:
Updated